sdk-src/src/test/java/com/hpe/adm/nga/sdk/unit_tests/common/CommonMethods.java

```diff
@@ -38,6 +38,10 @@ public static Octane getOctaneForTest() {
 
         return new Octane.Builder(new SimpleUserAuthentication("user", "password"), octaneHttpClient)
                 .Server(getDomain())
+                .settings(new Octane.OctaneCustomSettings() {{
+                    readTimeout = 120000;
+                    connectionTimeout = 2345;
+                }})
```

Comment:
Are the default settings not enough ? Do we have to set these here?

Reply:
no, i put them just to showcase the usage